tortoise/tortoise.go

```diff
@@ -540,31 +529,15 @@ func (t *turtle) processBallots(ctx *tcontext, ballots []*types.Ballot) error {
 				log.Named("last_evicted", t.LastEvicted))
 			continue
 		}
-		if b.LayerIndex().Before(lastLayerID) {
-			return errNotSorted
-		} else if b.LayerIndex().After(lastLayerID) {
-			lastLayerID = b.LayerIndex()
-		}
-
```

Comment:
double-checking. this check is removed because we don't call processBallots on late blocks anymore?

Reply:
pretty much, since we always sort layers with organizer i don't see in what case this check is useful